TreeDB/collections/column_physical_query.go

```
@@ -114,6 +120,126 @@ func (c *Collection) RunColumnPhysicalQuery(req ColumnPhysicalQueryRequest) (Col
+
+	type workerResult struct {
+		exec *columnPhysicalQueryExecutor
+		diag columnPhysicalScanDiagnostics
+		err  error
+	}
+	results := make([]workerResult, workers)
+	start := time.Now()
+	var cancel atomic.Bool
+	var wg sync.WaitGroup
+	for worker := 0; worker < workers; worker++ {
+		worker := worker
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "worker" can be deleted (Go 1.22+)

(copyloopvar)